spot_rl_experiments/spot_rl/envs/base_env.py

```diff
@@ -411,14 +413,16 @@ def step(  # noqa
         self.stopwatch.record("get_observations")
 
         self.num_steps += 1
-        print(
+        self.say(
             f"****************************************************num_steps: {self.num_steps}"
         )
         timeout = self.num_steps >= self.max_episode_steps
         if timeout:
-            print(f"Execution exceeded {self.max_episode_steps} steps. Timing out...")
+            self.say(
+                f"Execution exceeded {self.max_episode_steps} steps. Timing out..."
+            )
         else:
-            print(f"Execution has not exceeded {self.max_episode_steps} steps.")
+            self.say(f"Execution has not exceeded {self.max_episode_steps} steps.")
```

Comment:
There is no need for this change. Let's keep it `print`. No need to emit this as audio-out.